app/src/main/java/org/jellyfin/mobile/player/ui/PlayerMenus.kt

```diff
@@ -217,6 +243,18 @@ class PlayerMenus(
         subtitlesButton.setImageState(stateSet, true)
     }
 
+    private fun buildQualityMenu(menu: Menu, videoWidth: Int, videoHeight: Int) {
+        menu.clear()
+        val options = qualityOptionsProvider.getApplicableQualityOptions(videoWidth, videoHeight)
+        options.map { option ->
+            val title = when (val bitrate = option.bitrate) {
+                0 -> context.getString(R.string.menu_item_auto)
+                else -> "${option.maxHeight}p - ${formatBitrate(bitrate.toDouble())}"
```

Comment:
Do we want to use (untranslateable-) strings for these?

Reply:
Don't think it's necessary.